glib/src/key_file.rs

```diff
@@ -167,7 +203,7 @@ impl KeyFile {
 
     #[doc(alias = "g_key_file_get_string_list")]
     #[doc(alias = "get_string_list")]
-    pub fn string_list(&self, group_name: &str, key: &str) -> Result<Vec<GString>, Error> {
+    pub fn string_list(&self, group_name: &str, key: &str) -> Result<StrV, Error> {
```

Comment:
Should it return `(StrV, usize)` to get access to the length instead? There is an inconsistency here even before my patch with `keys()` and `groups()`: either we return StrV and size in all of these or we return Vec and omit size

Reply:
I went with the first option since I believe the codegen will generate that once it learns about strv

Reply:
I think you can generate a `glib::Slice<StrVItem>` or `glib::PtrSlice<StrVItem>` in this case?

Reply:
With either of those (and the difference is not very clear btw) I get

`the trait `translate::TransparentPtrType` is not implemented for `StrVItem``

Reply:
You could implement that though.

The difference is
  - `TransparentType`: A reference of the Rust type `&T` is equivalent to the native C pointer `*const ffi::T`
  - `TransparentPtrType`: The Rust type itself `T` is equivalent to the native C pointer `*const ffi::T`, or in other words `&T` is equivalent to `*const *const ffi::T`

The first is true for `glib::Value`, boxed inline, and such types, the second is true for GObjects, shared, boxed types.

As `StrVItem` is basically a `*const c_char` this means that it is a `TransparentPtrType`.